src/cc65/staticassert.c

```diff
@@ -0,0 +1,96 @@
+/*****************************************************************************/
+/*                                                                           */
+/*                               staticassert.h                              */
+/*                                                                           */
+/*          _Static_assert handling for the cc65 C compiler                  */
+/*                                                                           */
+/*                                                                           */
+/*                                                                           */
+/* Copyright 2020 Google LLC                                                 */
+/*                                                                           */
+/*                                                                           */
+/* This software is provided 'as-is', without any expressed or implied       */
+/* warranty.  In no event will the authors be held liable for any damages    */
+/* arising from the use of this software.                                    */
+/*                                                                           */
+/* Permission is granted to anyone to use this software for any purpose,     */
+/* including commercial applications, and to alter it and redistribute it    */
+/* freely, subject to the following restrictions:                            */
+/*                                                                           */
+/* 1. The origin of this software must not be misrepresented; you must not   */
+/*    claim that you wrote the original software. If you use this software   */
+/*    in a product, an acknowledgment in the product documentation would be  */
+/*    appreciated but is not required.                                       */
+/* 2. Altered source versions must be plainly marked as such, and must not   */
+/*    be misrepresented as being the original software.                      */
+/* 3. This notice may not be removed or altered from any source              */
+/*    distribution.                                                          */
+/*                                                                           */
+/*****************************************************************************/
```

Comment:
Maybe I'm getting this wrong, but at least as far as I understand we can't have a source file with such a copyright notice in the cc65 code base.

Reply:
I just copied it from another file; [nearly all seem to have it](https://github.com/cc65/cc65/search?q="This+notice+may+not+be+removed+or+altered+from+any+source+"&unscoped_q="This+notice+may+not+be+removed+or+altered+from+any+source+").

It's straight from the [zlib license that cc65 uses](https://github.com/cc65/cc65/blob/153bb295010e2dc75ffd544d7091870ff563338d/LICENSE).

Reply:
I'm pretty sure that no other file says `Copyright 2020 Google LLC`.

Reply:
Ah, that's not the part that looked like it was highlighted.

That's who owns the copyright, so it has to be like that if there's a new file. Is there an existing file it could be added to?

Most copyright notices say "Ullrich von Bassewitz", but there are quite a few with something else:
```
% find . -name '*.[ch]' | xargs egrep '\(C\) *[0-9]{2}' | grep -v Ullrich | head
./include/_6522.h:/* (C) 2004 Stefan Haubenthal <polluks@sdf.lonestar.org>                     */
./include/_antic.h:/* (C) 2000 Freddy Offenga <taf_offenga@yahoo.com>                           */
./include/_gtia.h:/* (C) 2000 Freddy Offenga <taf_offenga@yahoo.com>                           */
./include/_pbi.h:/* (C) 2000 Freddy Offenga <taf_offenga@yahoo.com>                           */
./include/_pia.h:/* (C) 2000 Freddy Offenga <taf_offenga@yahoo.com>                           */
./include/_pokey.h:/* (C) 2000 Freddy Offenga <taf_offenga@yahoo.com>                           */
./include/accelerator.h:/* (C) 2018 Marco van den Heuvel                                             */
./include/apple2.h:/* (C) 2000  Kevin Ruland, <kevin@rodin.wustl.edu>                           */
./include/apple2_filetype.h:/* (C) 2017  Bill Chatfield, <bill_chatfield@yahoo.com>                      */
./include/apple2enh.h:/* (C) 2004  Oliver Schmidt, <ol.sc@web.de>
```

Reply:
> Ah, that's not the part that looked like it was highlighted.

Sorry, my fault :-(

> Most copyright notices say "Ullrich von Bassewitz", but there are quite a few with something else: [...]

Sorry for being too imprecise where I'm heading to :-( I'd like you to use either your real name (or `Ullrich von Bassewitz` if you don't want to give it). I don't want to see any Company names or alike. I'm no lawyer, but I'm just not interested in thinking about the potential consequences of have something else beside a real name here.

**_And I certainly don't want anybody to discuss this matter with me !_**

Reply:
Am I right in presuming that my desire for a natural person as copyright holder is no showstopper?

Reply:
> Am I right in presuming that my desire for a natural person as copyright holder is no showstopper?

Not a showstopper, but it may make things more difficult depending on what you will allow there.

I work for Google, so they own the copyright. [Here are the requirements for copyright notices](https://opensource.google/docs/patching/#license-headers-and-copyright-notices). 

I could also say "Copyright 2020 The cc65 Authors". Is that ok with you?

If you really want a real person, then there's a [special process I have to go through](https://opensource.google/docs/iarc/).

Reply:
> I could also say "Copyright 2020 The cc65 Authors". Is that ok with you?

That would be great. Thanks for your understanding :-)

Reply:
Updated.